.vscode/launch.json

```diff
@@ -17,24 +17,26 @@
       "env": {
         "NODE_ENV": "development",
         "PORT": "4000",
-        "MONGO_URI": "mongodb://localhost:27017/meeorder"
+        "MONGO_URI": "mongodb://localhost:27017",
+        "MONGODB_NAME": "meeorder"
```

Comment:
According to your code in `features/step-definitions/health.ts`, shouldn't this be `MONGO_DB_NAME`?